### DISCUSSION TITLE
Nabu Casa (Home Assistant founders company) have joined the Z-Wave Alliance and plan on certifying Z-Wave JS

### DISCUSSION THREAD

**Discussion by Hedda** [zwave-js/zwave-js#6678] [General]
Awesome news! Nabu Casa has joined the Z-Wave Alliance and now has plans to certify Z-Wave JS as a certified Z-Wave solution:

https://www.home-assistant.io/blog/2024/02/15/nabu-casa-joins-z-wave-alliance/

"**_thanks to revenue from Home Assistant Cloud subscribers, and will start the certification process for Z-Wave JS_**"

https://www.nabucasa.com/about/

Be sure to sign up to Home Assistant Cloud to sponsor Nabu Casa employees development of Home Assistant and Z-Wave JS!

https://www.home-assistant.io/cloud/

**Comment by Hedda** [comment]
@AlCalzone follow-up question to this is if you can tell us if Nabu Casa also has a plan on making and selling an official Home Assistant Z-Wave 800 Series USB dongle hardware? 

I know that [Z-Wave 800 Series only has experimental cutting-edge support in Z-Wave JS](https://github.com/zwave-js/node-zwave-js/issues/5257), however, I believe a USB dongle solution based on something like the pre-certified Silicon Labs ZGM230S SiP based on Silabs EFR32ZG23 SoC would still be the perfect fit because Z-Wave 800 Series is the first generation of Z-Wave radio chips that have worldwide frequency support so the only thing that needs to be different is the firmware for different RF regions, (though may have to be pre-flashed before shipping for different RF regions be CE, FCC and ISED approved?).

Having an official Home Assistant branded Z-Wave 800 Series USB dongle hardware made by Nabu Casa in a similar form factor to the [Home Assistant SkyConnect](https://skyconnect.home-assistant.io/) would be awesome!

I think that [Stefan Agner (@agners)](https://github.com/agners) who like yourself is also an employee at Nabu Casa designed and developed the [Home Assistant SkyConnect](https://www.home-assistant.io/skyconnect) (as well as the [Home Assistant Yellow](https://www.home-assistant.io/yellow)) so wondering if you have already talked to him about you at Nabu Casa possibly designing and manufacturing Z-Wave Controller USB dongle that can be branded and sold as official Home Assistant hardware?

**Reply by AlCalzone** [reply]
> follow-up question to this is if you can tell us if Nabu Casa also has a plan on making and selling an official Home Assistant Z-Wave 800 Series USB dongle hardware?

I guess [now](https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead/) I can... Yes!

**Reply by Hedda** [reply]
https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead/

"**_Uwe leads the development of the Home Assistant Z-Wave stick that we’re working on_**"

Nice! That is indeed more awesome news! 

I also heard this mentioned on both "State of the Open Home 2024" and "Home Assistant 2024.5 release party" videos:

https://www.home-assistant.io/blog/2024/04/24/state-of-the-open-home-2024/

https://www.home-assistant.io/blog/2024/05/01/release-20245/

PS: Tip; include a longer USB extension cable in the package with instructions to use it for the best user-experience 😉

**Comment by Hedda** [comment]
Oh, and in late but related news; Silicon Labs (who makes all the Z-Wave chips) has also partnered with Nabu Casa to support open source:

https://www.home-assistant.io/blog/2023/12/12/silicon-labs-official-partnership-nabu-casa/
